### PR TITLE
WL-4756: Images in first column are tiny

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: mneme-project/ambrosia/ambrosia-impl/impl/src/java/org/etudes/ambrosia/impl/UiNavigation.java [hunk 1]
```diff
@@ -24,22 +24,17 @@
 
 package org.etudes.ambrosia.impl;
 
-import java.io.PrintWriter;
```
Comment (buckett) [inline mneme-project/ambrosia/ambrosia-impl/impl/src/java/org/etudes/ambrosia/impl/UiNavigation.java:27]: This shouldn't be in here.

File: mneme-project/ambrosia/ambrosia-library/lib/src/webapp/skin/ambrosia.css [hunk 2]
```diff
@@ -310,7 +310,10 @@ Sakai ~ itemAction
 	font-size: .8em;
 	white-space: nowrap;
 }
-
+.ambrosiaIcon
```
Comment (buckett) [inline mneme-project/ambrosia/ambrosia-library/lib/src/webapp/skin/ambrosia.css:313]: Would it be better to define this centrally and then then use that class in the places it's needed?
```
.nonResponsive {
   max-width: none;
}
```

Comment (nicholaswilson100) [reply mneme-project/ambrosia/ambrosia-library/lib/src/webapp/skin/ambrosia.css:313]: Maybe.  I'm not sure the number of cases so far warrants it does it?

Comment (buckett) [reply mneme-project/ambrosia/ambrosia-library/lib/src/webapp/skin/ambrosia.css:313]: Sure.

**Pull request merged**